Source/Blazorise/wwwroot/datePicker.js

```diff
@@ -74,7 +74,7 @@ export function initialize(dotnetAdapter, element, elementId, options) {
     if (options) {
         picker.altInput.disabled = options.disabled || false;
         picker.altInput.readOnly = options.readOnly || false;
-        picker.altInput.placeholder = options.placeholder;
+        picker.altInput.placeholder = options.placeholder !== null ? options.placeholder : "";
```

Comment:
I also like it.

I have used the coalescing naming for functions like these before, that return a default value on null. (comes from SQL I guess?) https://learn.microsoft.com/en-us/sql/t-sql/language-elements/coalesce-transact-sql?view=sql-server-ver16
So I think it should be something like `coalesce` or at least something that you can easily understand that it will default to some value on `Null`.

`isNull` naming would be for a boolean check.